src/nvim/ex_cmds.c

```diff
@@ -1071,6 +1071,15 @@ void do_bang(int addr_count, exarg_T *eap, bool forceit, bool do_in, bool do_out
   } else {                            // :range!
     // Careful: This may recursively call do_bang() again! (because of
     // autocommands)
```

Comment:
the comment is out of place now?

Reply:
Quite sure I did not change this now, I added some code below.